website/content/docs/upgrading/upgrade-to-1.8.0.mdx

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.8.0 - Guides
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.8.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.8.0 compared to 1.7. Please read it carefully.
+
```

Comment:
Sub-section here for Go version update, similar to https://www.vaultproject.io/docs/upgrading/upgrade-to-1.6.0#go-version?

Reply:
Good idea